ibm/service/iampolicy/resource_ibm_iam_policy_assignment.go

```diff
@@ -576,3 +553,126 @@ func ResourceIBMPolicyAssignmentAssignmentTemplateDetailsToMap(model *iampolicym
 	}
 	return modelMap, nil
 }
+
+func waitForAssignment(timeout time.Duration, meta interface{}, d *schema.ResourceData, refreshFn func(string, interface{}) resource.StateRefreshFunc) (interface{}, error) {
```

Comment:
Do we want terraform provider (or clients in general )waiting for assignments?

I see Delay and Timeout in other terraform resources, so I think it is okay. I'm curious.